src/components/ChoiceList/ChoiceList.tsx

```diff
@@ -19,6 +20,8 @@ export interface ChoiceDescriptor {
   disabled?: boolean;
   /** Additional text to aide in use */
   helpText?: React.ReactNode;
+  /** Indicates that the choice is aria-describedBy the error message*/
+  describedByError?: boolean;
```

Comment:
The reason I went with `describedByError: boolean` as oppose to `describedBy:string` is because ChoiceList is the one generating the error and the id. So I thought to add `describedByError`  to the descriptor was more clear. The choice has 1 error and this choice is described by that error.